test/e2e/conftest.py

```diff
@@ -13,15 +13,55 @@
 
 import pytest
 import boto3
+import logging
+import time
+from dataclasses import dataclass
 
-from acktest.aws.identity import get_region
-from acktest import k8s
+from acktest.resources import random_suffix_name
+from acktest.aws.identity import get_region, get_account_id
+from acktest import k8s as k8sclient
+from acktest.k8s import resource as k8s
 
+from e2e import CRD_GROUP, CRD_VERSION, load_lambda_resource
+from e2e.service_bootstrap import LAMBDA_FUNCTION_FILE_ZIP
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.bootstrap_resources import get_bootstrap_resources    
+
+FunctionPackageTypeImage = "Image"
+FunctionPackageTypeZip = "Zip"
+FunctionTestingRoleBasic = "basic"
+FunctionTestingRoleESM = "esm"
+
+@dataclass
+class ResourceWaitTimes:
+    Create: int
+    Update: int
+    Delete: int
```

Comment:
Could be defined in the `test-infra` package. Also, we should probably provide sane defaults (say 5 seconds for each?)